test/persona_client_test.js

```diff
@@ -757,6 +757,116 @@ describe("Persona Client Test Suite", function(){
             });
         });
 
+       it("should correctly validate an invalid scoped token - Persona route version 1 - should return 401", function(done){
+            // use _getOAuthToken to generate a token outside of the node client
+
+            _getOAuthToken("primate", function(err, token){

```

Comment:
Can you add to these tests, in case of an error getting the token:
```javascript
if(err) {
  done(err);
  return;
}
```
This is so that failure messages better pin point root causes.